### PR TITLE
Curation recruitment announcement

### PR DESCRIPTION
**Pull request by TheCzar1994**

### CODE REVIEW

File: src/collections/posts/curator-applications-are-now-open.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+title: Curator applications are currently open!
+section: announcements
+category: announcement
+publish: 2024-04-07T18:37:19.983Z
+image: /static/beastsaber-logo-fullsize-square.jpg
+showInPostListing: false
+---
+# Curation Recruitment is Currently Open!
+
+**We are re-opening our curator applications! We’re looking for 3 outstanding applicants of any skill level. These applicants should be people who:**
```
Comment (TimothyGillespie) [inline src/collections/posts/curator-applications-are-now-open.md:11]: The ' on we're seems to be the "fancy" one again - if you wanted to change that.

File: src/collections/posts/curator-applications-are-now-open.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+- have plenty of free time, enthusiasm, and passion for playing and curation,
+
+- are able to demonstrate familiarity with the “Objective Issues” which bar a map from curation as listed in our BeastSaber Curation Team Handbook,
```
Comment (TimothyGillespie) [inline src/collections/posts/curator-applications-are-now-open.md:15]: Seems like it could link the HandBook here quite nicely.

**Review by TimothyGillespie — APPROVED** [review]
Looks good to me and works. Announcement seems to be working as it should and look generally good. Linking the picture like this should be fine too - could create some issues when editing in the CMS perhaps, but I'll check that separately. 

Just remarks as usual that you can address as you see fit - so no requested changes or similar. So approval from my side.

I did btw notice the Google Form seems to be a bit outdated on the Discord Tag. It reads:

> BeastSaber Username | Discord tag (i.e., HelenCarnate | helencarnate#2079)

Since the denominator is no longer a thing that might be confusing.

File: src/collections/posts/curator-applications-are-now-open.md (hunk 1 above)
Comment (TheCzar1994) [reply src/collections/posts/curator-applications-are-now-open.md:11]: ~~Maybe I just like my fancy apostrophes!~~
I will update it

File: src/collections/posts/curator-applications-are-now-open.md (hunk 2 above)
Comment (TheCzar1994) [reply src/collections/posts/curator-applications-are-now-open.md:15]: I considered this, but it is linked just below, albeit, with different verbiage as "Curation requirements and expectations"... One day we'll update our standard announcement. Guess no harm in linking twice though.

**Pull request merged**